spec/rubocop/cli/cli_autocorrect_spec.rb

```diff
@@ -1653,6 +1653,32 @@ def self.some_method(foo, bar: 1)
     RUBY
   end
 
+  it 'corrects Style/BlockDelimiters offenses when specifing' \
```

Comment:
small typo here (if you care): "specifying" instead of "specifing"

Thanks so much for posting the bugfix in such a timely manner!  I am amazed at how quickly you did so!  You're awesome!

Reply:
This typo fixed by https://github.com/rubocop-hq/rubocop/commit/e97eb5ce74b3319c21de05f6c1fd78493bd55b47. Thanks!